tests/unit/lms/services/mailchimp_test.py

```diff
@@ -61,7 +120,7 @@ def test_it_doesnt_send_duplicate_emails(
                 "lms:templates/email/instructor_email_digest/",
                 sender,
                 recipient,
-                template_vars={},
+                template_vars={"total_annotations": 1},
```

Comment:
The new `{% if %}` in the template means that `total_annotations` is needed in the template variables otherwise rendering the template crashes. But none of the other variables are needed.